src/aura/CMP_Acc_Naming_Dropdown/CMP_Acc_Naming_DropdownHelper.js

```diff
@@ -1,4 +1,10 @@
 ({
+    //@TODO: Need to refactor this method to avoid calling each of these two methods twice
```

Comment:
@lifewithryan , I think it's because this branch is old and if it's a draft branch it doesn't get pulled down? I don't know why Xinnan's update didn't make it to this draft.